build-tools/src/main/kotlin/org/jetbrains/kotlin/XcRunRuntimeUtils.kt

```diff
@@ -0,0 +1,63 @@
+package org.jetbrains.kotlin
+
+import kotlinx.serialization.Serializable
```

Comment:
Hmm, is it indeed a good idea to depend on `kotlinx.serialization.` here?

Reply:
We already have dependencies on it in `build-utils` (see `ExternalReportUtils.kt`) and I don't think it can cause any trouble.